web/themes/custom/usagov/templates/views-view-list--sub-children.html.twig

```diff
@@ -20,75 +20,94 @@
-{% for row in rows %}
-<li class="usagov-navpage-item">
-  <a href="{{ drupal_url('node/' ~ row.content["#row"].nid) }}" class="text-no-underline">
-    {{- row.content -}}
-  </a>
-</li>
-{% endfor %}
-</ul>
+	<ul class="usa-list usa-list--unstyled">
+		{% for row in rows %}
+			<li class="usagov-navpage-item">
+				<a href="{{ drupal_url('node/' ~ row.content["#row"].nid) }}" class="text-no-underline">
+					{{- row.content -}}
+				</a>
+			</li>
+		{% endfor %}
+	</ul>
 
 {% elseif term_name == "Navigation Cards Page" or term_name == "Life Events" or term_name == "Life Events Landing Page" %}
 
-<div class="usagov-cards margin-top-3">
-  <ul class="usa-card-group">
-    {% for row in rows %}
-    <li class="tablet:grid-col-4 usa-card">
-      <a href="{{ drupal_url('node/' ~ row.content["#row"].nid) }}" class="usa-card__container usagov-card text-no-underline">
-      {% if term_name == "Life Events Landing Page" %}
-       <div class="life-events-landing-page-card-image" style="background-image: url({{file_url(row.content["#row"]._entity.field_navigation_banner_image.entity.field_media_image.entity.fileuri)}})"></div>
-       <div class="life-events-landing-page-card-title">{{row.content["#row"].node_field_data_title}}</div>
-      {% else %}
-        {{- row.content -}}
-      {% endif %}
-      </a>
-    </li>
-    {% endfor %}
-  </ul>
-</div>
-{% if lang == "en" %}
-<p><span class="text-bold">Looking for something else?</span> <a id="something-else" href="/#all_topics">Explore all topics and services</a></p>
-{% else %}
-<p><span class="text-bold">¿Busca algo más? <a id="something-else" href="/es#all_topics">Encuentre más temas y servicios</a></span></p>
-{% endif %}
+	<div class="usagov-cards margin-top-3">
+		<ul class="usa-card-group">
+			{% for row in rows %}
+				<li class="tablet:grid-col-4 usa-card">
+					<a href="{{ drupal_url('node/' ~ row.content["#row"].nid) }}" class="usa-card__container usagov-card text-no-underline">
+						{% if term_name == "Life Events Landing Page" %}
+							<div class="life-events-landing-page-card-image" style="background-image: url({{file_url(row.content["#row"]._entity.field_navigation_banner_image.entity.field_media_image.entity.fileuri)}})"></div>
+							<div class="life-events-landing-page-card-title">{{row.content["#row"].node_field_data_title}}</div>
+						{% else %}
+							{{- row.content -}}
+						{% endif %}
+					</a>
+				</li>
+			{% endfor %}
+		</ul>
+	</div>
+	{% if lang == "en" %}
+		<p>
+			<span class="text-bold">Looking for something else?</span>
+			<a id="something-else" href="/#all_topics">Explore all topics and services</a>
+		</p>
+	{% else %}
+		<p>
+			<span class="text-bold">¿Busca algo más?
+				<a id="something-else" href="/es#all_topics">Encuentre más temas y servicios</a>
+			</span>
+		</p>
+	{% endif %}
+
+{% elseif term_name == "Home Page" %}
 
-{% elseif term_name == "Standard Page" %}
-{# standard_page #}
+	{% if view.current_display == "block_4" %}
```

Comment:
Would it be too late to change "block_4" to a friendlier name (like "Intro"?) Drupal has squirreled that away in the "Advanced" section of the block definition.

Reply:
So these are what it was before I started on the homepage. I am not sure why they were named that way. You'll see for the life-events view I did rename it but I didn't want to touch the other blocks.

Reply:
So these are what it was before I started on the homepage. I am not sure why they were named that way. You'll see for the life-events view I did rename it but I didn't want to touch the other blocks. 

I'm looking at the code again to make sure that it isn't used anywhere else but I should be able to rename it

Reply:
Renamed!